components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/docs/examples/apis/validate_openapi_post.yaml

```diff
@@ -0,0 +1,212 @@
+- title: Using a Valid OpenAPI Definition
+  rawRequest: |
+    POST https://localhost:9443/api/am/publisher/v1.0/apis/validate-openapi?returnContent=true HTTP/1.1
+    Content-Type: multipart/form-data; boundary="----=_Part_0_771605626.1563876237955"
+    Authorization: Bearer 031cf7d6-4f49-3089-a164-5289b171203a
+    Content-Length: 1504
+
+    ------=_Part_0_771605626.1563876237955
+    Content-Type: application/octet-stream; name=swagger2-sample.1.0.0.yaml
```

Comment:
these headers should go to the rawResponse?

Reply:
You mean `rawRequest`? This is the raw request I extracted from SoapUI. I think Individual item's content type should present when sending a multipart request with files.